modules/gapi/test/common/gapi_core_tests_inl.hpp

```diff
@@ -1267,6 +1267,193 @@ TEST_P(NormalizeTest, Test)
     }
 }
 
+TEST_P(BackendOutputTest, EmptyOutput)
```

Comment:
BackendOutput -> BackendOutputAllocation (just to make it clear)

Reply:
agree, sounds better, updated